test/components/AuthForm_test.js

```diff
@@ -241,57 +241,31 @@ describe("AuthForm component", () => {
         ],
         session: { authenticated: false, serverInfo: DEFAULT_SERVERINFO },
       };
-      const wrapper = mount(<AuthForm {...props} />);
-      expect(wrapper.find("input#root_server").prop("value")).eql(
+      const wrapper = render(<AuthForm {...props} />).container;
+      expect(wrapper.querySelector("input#root_server").value).eql(
         "http://server.test/v1"
       );
-      expect(wrapper.find("input#root_authType").prop("value")).eql(
+      expect(wrapper.querySelector("input#root_authType").value).eql(
         "basicauth"
       );
 
       // Changing the server to another element from the servers history.
-      wrapper.find("input#root_server").simulate("change", {
+      fireEvent.change(wrapper.querySelector("input#root_server"), {
         target: { value: "http://test.server/v1" },
       });
-      expect(wrapper.find("input#root_server").prop("value")).eql(
+      expect(wrapper.querySelector("input#root_server").value).eql(
         "http://test.server/v1"
       );
       // authType is reset to "anonymous" while we wait for the server info
       // (capabilities)
-      expect(wrapper.find("input#root_authType").prop("value")).eql(
+      expect(wrapper.querySelector("input#root_authType").value).eql(
         "anonymous"
       );
 
-      // Simulate a `getServerInfo` response that updated the `capabilities`.
-      const prevProps = wrapper.props();
-      wrapper.setProps({
-        ...prevProps,
-        session: {
-          ...prevProps.session,
-          serverInfo: {
-            ...prevProps.session.serverInfo,
-            capabilities: {
-              ...prevProps.session.serverInfo.capabilities,
-              basicauth: "some basic auth info",
-              ldap: "some ldap auth info",
-              fxa: "some fxa auth info",
-              openid: {
-                providers: [
-                  {
-                    name: "google",
-                  },
-                ],
-              },
-            },
-          },
-        },
-      });
-      // authType is now set to the value we have from the servers history.
-      // TODO: for some reason, we don't have a "input#root_authType" anymore
-      // at this point, so we need to get at the `authType` value some other
-      // way, here by looking into the state.
-      const state = wrapper.find(AuthForm).instance().state;
-      expect(state.formData.authType).eql("openid-google");
+      // TODO: Simulate a `getServerInfo` response that updated the `capabilities`.
+      // The enzyme test has been removed as part of the upgrade to testing-library,
+      // but not replaced because it interracted with properties and state directly.
+      // This needs to be rewritten to mock user input and server response instead.
```

Comment:
Have you looked into how much work it'd take to write this test in a `testing-library`-friendly way? It'd be nice to keep this test coverage rather than delete the test.

Reply:
Attempted to add a test that captures the spirit of the old test. Let me know your thoughts on this.

Reply:
Lgtm! A pragmatic workaround.